server/api_router.go

```diff
@@ -148,6 +148,18 @@ func (p *Plugin) newAPIRouter() *mux.Router {
 		}
 	}).Methods("POST")
 
+	// Stats
+	router.HandleFunc("/stats", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Considering the weight of the call, should we at least rate limit this pretty heavily?

Reply:
We could, but the global rate limiter we have on API endpoints would already apply to this admin-only endpoint coming from a pretty specific page, so I am not sure if the extra complication is worth it.